src/util.hpp

```diff
@@ -496,8 +496,7 @@ inline bool contains(const Container & container, const Value & value)
 
 
 #if 1
-# include <SDL_types.h>
-typedef Sint32 fixed_t;
+typedef int32_t fixed_t;
```

Comment:
Yes it is meant to be used as a fixed point real number. As you can see these is also a 
 typedef float fixed_t; in the #else branch.

I personally don't think this discussion is related to this pr which just replaces Sint32  wuth int32_t to remove a sdl dependency.